### PR TITLE
13092023 mahi

### PR DESCRIPTION
**Pull request by mahibashardeveloper**

### CODE REVIEW

File: app/Http/Controllers/FrontController.php [hunk 1]
```diff
@@ -9,4 +9,12 @@ class FrontController extends Controller
     public function home(){
         return view('pages.home');
     }
+
+    public function homePage_with_background_video(){
+        return view('pages.homePage-with-background-video');
+    }
+
+    public function homePage_with_background_parallaxEffect(){
```
Comment (rdhafiz) [inline app/Http/Controllers/FrontController.php:17]: Rename function name to homePage_with_background_parallax_effect

File: resources/scss/componets/04_backgrounds.scss [hunk 2]
```diff
@@ -19,3 +19,20 @@
...
+.background-parallax {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    background-size: cover;
+    background-position: center;
+    transform: translateY(0);
+}
```
Comment (rdhafiz) [inline resources/scss/componets/04_backgrounds.scss:38]: Write this by following scss parent child rules

**Review by rdhafiz — CHANGES_REQUESTED** [review]

File: app/Http/Controllers/FrontController.php (hunk 1 above)
Comment (mahibashardeveloper) [reply app/Http/Controllers/FrontController.php:17]: i rename this text

File: resources/scss/componets/04_backgrounds.scss (hunk 2 above)
Comment (mahibashardeveloper) [reply resources/scss/componets/04_backgrounds.scss:38]: i write this code in scss parent child rules

**Review by rdhafiz — APPROVED** [review]

**Review by rdhafiz — APPROVED** [review]

**Pull request merged**